mscore/scoreview.cpp

```diff
@@ -1990,6 +1994,8 @@ void ScoreView::cmd(const char* s)
                   }
             else {
                   Element* ele = _score->move(cmd);
+                  if (cmd == "empty-trailing-measure")
+                        changeState(ViewState::NOTE_ENTRY);
```

Comment:
Again, it made the most sense to enter into NoteEntry after doing this command because it is for the explicit purpose of going to an empty measure, and the odds are that this is a preparation for continuing note entry.